code-examples/protect-page-login/expressjs/index.js

```diff
@@ -0,0 +1,22 @@
+const app = require("express")()
+// highlight-next-line
+const sdk = require("@ory/client-fetch")
+
+// highlight-start
+const ory = new sdk.FrontendApi(
+  new sdk.Configuration({
+    basePath: process.env.ORY_ENDPOINT || "http://localhost:4000/.ory",
```

Comment:
We use `ORY_SDK_URL` consistently across the docs, can you use that here too please? :)